Added blog post about Survey 2022 (#296)
strimzi/strimzi.github.io: _posts/2022-05-13-survey-2022.md

---
layout: post
title: "Strimzi survey: we want to hear from you!"
date: 2021-05-13
author: paolo_patierno
---

> As every year, with the help of the CNCF, we run a survey of the Strimzi community.

Reviewer: The last one was Aug/Sept 2020, so maybe something like this:

```suggestion
With the help of the CNCF, we're running another survey of the Strimzi community.
```

> The main purpose is about hearing from the users in order to know how they are using Strimzi.

Reviewer: ```suggestion
The main purpose is to hear from Strimzi users in order to know how they are using Strimzi.
```

This is really useful to the maintainers to make better decisions in terms of new features and define the roadmap of the project itself.

> It is a really short survey with a few questions but it represents a huge help for us.
> It would be great if you could take the time for it.

Reviewer: ```suggestion
It would be great if you could take the 5–10 minutes needed to answer the 10 questions.
```

<!--more-->

> ## What we are interested in?

Reviewer: It's not a direct question, so no `?`
```suggestion
## What we are interested in
```

> We are interested to know what is your usage of Strimzi, if just for development, evaluation or already in production.

Reviewer: ```suggestion
We are interested in knowing about your usage of Strimzi, whether it's for development, evaluation or already in production.
```

> In order to provide better support and backward compatibility, it is important to get the most used Kubernetes versions.

Reviewer: ```suggestion
In order to provide better support and backward compatibility, it is important to have an understanding of what Kubernetes versions our users are deploying Strimzi on.
```

Kubernetes API changes and deprecations can have an impact on the Strimzi project as well.

> Upgrading a Strimzi cluster can be done at different times by different users.
> On every new operator release, when new valuable features are added or when a new Kafka version is supported. What is your choice?

Reviewer: ```suggestion
Similarly, it's helpful to the project to understand how quickly users adopt new Strimzi releases, and what prompts them to upgrade.
```

> One of the most recent features was the Kafka Connect Build.

Reviewer: Can we link to a blog or video about this?

Author: Yeah, I added a link to this one https://strimzi.io/blog/2021/03/29/connector-build/

It allows to build a new Kafka Connect image including the connectors you need, from different sources.
Maybe not all the users in the community know about it or others are already using it.
It is really useful for us to know.

We also introduced several Feature Gates in the recent versions.

> You can enable and try new functionalities thanks to the gates, to test what is new before it will be enabled by default.

Reviewer: ```suggestion
These gates enable you to try new functionality and test it before it gets enabled by default.
But right now we don't know how many users are actually making use of them. 
```

There are different ways to be engaged within the community and with the maintainers directly.

> If you use the Slack channel, the mailing list or GitHub discussions is useful to know for us.
> Reading the documentation, blog posts or the source code can be valuable to solve your issues. How much you use these different ways?

Reviewer: ```suggestion
Reading the documentation, blog posts or the source code can be valuable to solve your issues. But sometime users require more interactive help. So we'd like to know which of the these options people find most useful when learning about how to use Strimzi effectively.
```

Regarding our blog, we would like to know what are the topic of most interest to you.
We want to write useful content which could help the community to use the project or to learn more about Apache Kafka as well.

> If you are willing to take it, [here](LINK_PLACEHOLDER) is the survey that will be opened until <DEADLINE_PLACEHOLDER>.

Reviewer: ```suggestion
If you are willing to take it, [here](LINK_PLACEHOLDER) is the survey that will be open until <DEADLINE_PLACEHOLDER>.
```
